src/scripts.js

```diff
@@ -142,40 +140,98 @@ function buildTags(recipe) {
     .join(' ');
 }
 
+function displayAllTags() {
+  let allTags;
+
+  if (allRecipesTab.checked) {
+    allTags = recipeRepo.retrieveAllTags();
+  }
+  else if (!allRecipesTab.checked) {
+    allTags = user.favoriteRecipeRepo.retrieveAllTags();
+  }
+
+  allTags.forEach((tag) => {
+    const tagElement = document.createElement('p');
+    tagElement.classList.add('recipe-tag');
+    tagElement.innerText = tag;
+
+    tagElement.onclick = tagsToggleFilter;
+
+    tagsContainer.appendChild(tagElement);
+    tagList.push(tagElement);
+  });
+}
+
 function tagsToggleFilter(event) {
```

Comment:
I like the split up of the big `tagsToggleFilter` into a smaller function with helper functions. Much more readable!